taqueria-protocol/ParsedPluginInfo.ts

```diff
@@ -1,29 +1,10 @@
 import createType from '@taqueria/protocol/Base';
-import * as ParsedOperation from '@taqueria/protocol/ParsedOperation';
 import * as PluginInfo from '@taqueria/protocol/PluginInfo';
 import { z } from 'zod';
 
-const internalSchema = PluginInfo.internalSchema.extend({
-	operations: z.preprocess(
-		val => val ?? [],
-		z.array(
-			ParsedOperation.schemas.schema,
-			{ description: 'ParsedOperations' },
-		)
-			.optional(),
-	),
-}).describe('ParsedPluginInfo');
+const internalSchema = PluginInfo.internalSchema.extend({}).describe('ParsedPluginInfo');
```

Comment:
Why are we deleting or trimming all these `zod` types in this PR?

Reply:
We're removing operations. They're hard to differentiate from tasks, and I've found the real underlying difference is context. So instead of having a separate construct/type called "operation", I'm reusing tasks but in a different context.